tests/test_task.py

```diff
@@ -159,19 +159,6 @@ def test_version_loaded(self):
         except Exception:
             self.fail('Could not load the taskgraph version as expected.')
 
-    def test_version_not_loaded(self):

```

Comment:
Why remove this test?

Reply:
@phargogh and I discussed this. This test depended on `mock` and would only fail `taskgraph` were incorrectly installed -- perhaps by someone copying the source tree. James originally wrote the test and said he'd never experienced it otherwise and the case of a broken `__version__` due to user error seemed like something not worth guarding against... Otherwise what else would you check for? A user deleting a random file? A bug in Python? etc...